tests/scripts/checkFileSize.js

```diff
@@ -0,0 +1,53 @@
+const fs = require( 'fs' );
+const path = require( 'path' );
+
+const sizeLimits = {
+	'frontend.css': 73,
+	'frontend.min.css': 63,
```

Comment:
Can we set the limit to the current file size?

Reply:
@Svitlana-Dykun

Reply:
@rami-elementor Done, set to: 
	'frontend.css': 64,
	'frontend.min.css': 54,